### ISSUE TITLE
Send email in text format -> no text only logo file visible in message

### ISSUE THREAD

**Issue by ProBackup-nl** [PrestaShop/PrestaShop#14456]
**Describe the bug**
Text content of sent email messages are not visible in Thunderbird, only the logo file. In view source the message text part is visible.

**To Reproduce**
1. Go to '[BO] > Advanced Parameters > E-mail'
2. Choose 'Send email in text format' and 'Use PHP's mail() function (recommended; works in most cases)'
3. Configure a logo in '[BO] Design > Theme & Logo > Invoice & email logos' 
4. Now sent a message, like 'Welcome new customer' (during order creation, with allow guest disabled)
5. See Thunderbird, only logo is shown no text
6. I expect only text visible and no logo (attachment)

**Screenshots**
![Welkom_Zonnehuukske](https://user-images.githubusercontent.com/515451/60516492-29d4de80-9cde-11e9-8c01-78b32c251cf9.png)

**Message source**
```
<snip>
Subject: [`t Zonnehuukske] Welkom!
X-PHP-Script: <snip>/index.php for <snip>
X-PHP-Filename: <snip> REMOTE_ADDR: <snip>
Message-ID: <20190702075126.161920683.swift@zonnehuukske.nl>
Date: Tue, 02 Jul 2019 09:51:26 +0200
From: `t Zonnehuukske <snip>
Reply-To: <snip>
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="_=_swift_v4_1562053886_8a8ffe8b710e1c7311078922ecbefc1b_=_"


--_=_swift_v4_1562053886_8a8ffe8b710e1c7311078922ecbefc1b_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable


[https://zonnehuukske.nl/nl/]=20

Hallo Voornaam Achternaam,

Beda=
nkt voor het aanmaken van uw klantaccount bij `t Zonnehuukske.=20

Hier z=
ijn uw aanmeldgegevens

E-MAILADRES: <snip> =09=09=20

Bel=
angrijke veiligheidstips

* Bewaar uw accountgegevens op een veilige pl=
aats.

* Verstrek uw aanmeldgegevens nooit aan derden.

* Wijzig uw=
 wachtwoord regelmatig.

* Als u vermoedt dat iemand anders uw account =
zonder uw toestemming
gebruikt, neem dan onmiddellijk contact met ons op.=


U kunt nu bestellingen op onze website plaatsen: `t Zonnehuukske
[h=
ttps://zonnehuukske.nl/nl/]=20

`t Zonnehuukske [https://zonnehuukske.nl/=
nl/] powered by
PrestaShop(tm) [http://www.prestashop.com/]=20



--_=_swift_v4_1562053886_8a8ffe8b710e1c7311078922ecbefc1b_=_
Content-Type: multipart/related;
 boundary="_=_swift_v4_1562053886_d05e78cd34cfc7398c6c93f5a79e9945_=_"


--_=_swift_v4_1562053886_d05e78cd34cfc7398c6c93f5a79e9945_=_
Content-Type: image/jpeg; name=my-shop-logo-1542149064.jpg
Content-Transfer-Encoding: base64
Content-ID: <1909a510be07accfda4b7ff90d0c9e27@zonnehuukske.nl>
Content-Disposition: inline; filename=my-shop-logo-1542149064.jpg

iVBORw0KGgoAAAANSUhEUgAAAT8AAABkCAYAAADwmDcrAAAgAElEQVR42uydeZwcVb32v+dU9Tr7
<snip>
AhcIncxod74jAAAAAElFTkSuQmCC

--_=_swift_v4_1562053886_d05e78cd34cfc7398c6c93f5a79e9945_=_--


--_=_swift_v4_1562053886_8a8ffe8b710e1c7311078922ecbefc1b_=_--


```
**Additional information**
PrestaShop version: 1.7.5.2
PHP version: 7.2.18
Theme: classic
Thunderbird: 60.7.2 (64-bit)

**Comment by khouloudbelguith** [comment]
Hi @ProBackup-nl,

I did not manage to reproduce the issue with PS1.7.6.0rc1 & the PS1.7.5.2.
![image](https://user-images.githubusercontent.com/16067358/60519192-f775b200-9cda-11e9-87a8-a88244bde9df.png)
![image](https://user-images.githubusercontent.com/16067358/60525336-13cb1c00-9ce6-11e9-8a92-91e2e1e65ab9.png)

Could you please provide us with more info? We need more details to understand how we can reproduce your issue:
 - server setup and configuration
 - PrestaShop version (source)
 - debug mode report
 - PHP error logs
 - apache error log
 - javascript console log

Don't you know how to get this information? Please read the following article:
http://build.prestashop.com/howtos/misc/how-to-create-bug-report/

Thanks!

**Comment by ProBackup-nl** [comment]
@khouloudbelguith Your message printscreen shows html formatting. Did you set `[BO] > Advanced Parameters > E-mail > (⬤) Send email in text format`?

**Comment by ProBackup-nl** [comment]
Choosing text format out of the options (html/text/both) results in not wishing to send an alternative format of the content. Therefore using "`multipart/alternative`" is already a problem in my opinion. Only use "`multipart/mixed`" (text with attachments) in such case. Not wrapped inside "`multipart/alternative`".

**Comment by khouloudbelguith** [comment]
@ProBackup-nl, thanks for your feedback.
Yes, you are right, I have the same issue with PS1.7.5.2 & PS1.7.6.0rc1.
I tried with different email: Welcome module, order confirmation email=> only the logo is displayed.
![image](https://user-images.githubusercontent.com/16067358/60585854-93112c00-9d88-11e9-9136-40855c1c6ace.png)
![image](https://user-images.githubusercontent.com/16067358/60586020-074bcf80-9d89-11e9-9a81-d490a5d74bda.png)

I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!
Thanks!

**Comment by hibatallahAouadni** [comment]
reproduced on 1.7.8.2

**Comment by cruftex** [comment]
This is a dup of: 
https://github.com/PrestaShop/PrestaShop/issues/19070

I created a pull request:
https://github.com/PrestaShop/PrestaShop/pull/32155

**Comment by florine2623** [comment]
It is fixed in 8.1.x and 9.0.x.

The emails only contains text :

<img width="416" alt="Screenshot 2024-01-03 at 17 44 19" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/32749580-450e-4902-a480-c9e6839d0557">
<img width="251" alt="Screenshot 2024-01-03 at 17 44 33" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/bd5b3114-3dfb-401d-9206-1410cfcc807d">
<img width="1061" alt="Screenshot 2024-01-03 at 17 44 41" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/6b8de125-f99b-4934-bef0-493f120039f7">
<img width="903" alt="Screenshot 2024-01-03 at 17 45 03" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/962ec7a7-3c8a-4962-bbf8-87c4e6c42d3a">
